src/bp/core/services/hook/hook-service.ts

```diff
@@ -77,6 +77,12 @@ export namespace Hooks {
       super(1000, { bp, incident }, 'on_incident_status_changed')
     }
   }
+
+  export class BeforeBotImport extends BaseHook {
+    constructor(bp: typeof sdk, botId: string, tmpFolder: string, hookResult: object) {
+      super(1000, { bp, botId, tmpFolder, hookResult }, 'before_bot_import')
+    }
```

Comment:
I know that's not part of this PR but why do we pass the timeout as the first argument as it's clearly an option and could set a default arg. In the way we could remove the `1000` everywhere

Reply:
Good question, it bothers me also. ill change it